csharp/src/Ice/Internal/RouterInfo.cs

```diff
@@ -295,29 +287,12 @@ public RouterInfo get(Ice.RouterPrx rtr)
         }
     }
 
-    //
-    // Returns router info for a given router. Automatically creates
-    // the router info if it doesn't exist yet.
-    //
-    public RouterInfo erase(Ice.RouterPrx rtr)
```

Comment:
I'm guessing this comment was inaccurate.

Reply:
It was.